journald/lib_journald.hpp

```diff
@@ -41,6 +136,25 @@ struct Flags : public virtual flags::FlagsBase
           return None();
         });
 
+    add(&Flags::logrotate_path,
+        "logrotate_path",
+        "If specified, the logrotate container logger will use the specified\n"
+        "'logrotate' instead of the system's 'logrotate'.",
+        "logrotate",
+        [](const std::string& value) -> Option<Error> {
+          // Check if `logrotate` exists via the help command.
+          // TODO(josephw): Consider a more comprehensive check.
+          Try<std::string> helpCommand =
+            os::shell(value + " --help > /dev/null");
```

Comment:
Nope, these flags can only be changed by the operator (in `/opt/mesosphere/etc/...`)